src/effects/RealtimeEffectState.h

```diff
@@ -28,14 +30,15 @@ class RealtimeEffectState
    : public XMLTagHandler
    , public std::enable_shared_from_this<RealtimeEffectState>
    , public SharedNonInterfering<RealtimeEffectState>
+   , public ClientData::Site<RealtimeEffectState>
 {
 public:
    struct AUDACITY_DLL_API EffectFactory : GlobalHook<EffectFactory,
       const EffectInstanceFactory *(const PluginID &)
    >{};
 
    explicit RealtimeEffectState(const PluginID & id);
-   RealtimeEffectState(const RealtimeEffectState &other);
+   RealtimeEffectState(const RealtimeEffectState &other) = delete;
```

Comment:
Deletion of the copy constructor might be done in the commit "RealtimeEffectList performs shallow copy for states"